mindsdb/integrations/handlers/reddit_handler/reddit_handler.py

```diff
@@ -0,0 +1,296 @@
+import praw
+import pandas as pd
+from datetime import datetime
+from mindsdb.integrations.libs.api_handler import APIHandler, APITable
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.utilities.config import Config
+from mindsdb.utilities import log
+from mindsdb_sql.parser import ast
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+from mindsdb.integrations.utilities.date_utils import parse_utc_date
+
+from mindsdb_sql.parser.ast import Select
+
+
+class CommentTable(APITable):
```

Comment:
Can we please move this class to a new file reddit_tables.py and keep only the handler here?